src/test/java/api/requests/RequestTypeItemStatusWhiteListTests.java

```diff
@@ -122,4 +122,16 @@ public void cannotCreateRequestWhenItemStatusClaimedReturned(String requestType)
     assertFalse(RequestTypeItemStatusWhiteList
       .canCreateRequestForItem(ItemStatus.CLAIMED_RETURNED, RequestType.from(requestType)));
   }
+
+  @Test
+  @Parameters({
+    "",
+    "Hold",
+    "Recall",
+    "Page"
+  })
+  public void cannotCreateRequestWhenItemStatusWithdrawn(String requestType) {
+    assertFalse(RequestTypeItemStatusWhiteList
+      .canCreateRequestForItem(ItemStatus.WITHDRAWN, RequestType.from(requestType)));
+  }
```

Comment:
Could these be unit like tests rather than API tests?

Reply:
This class has been moved to `org.folio.circulation.domain` package. It is not part of API tests anymore.